TransIT.BLL/Helpers/Roles.cs

```diff
@@ -1,13 +1,13 @@
 namespace TransIT.API.Extensions
 {
-    public static class ROLE
+    public static class RoleConsts
```

Comment:
- remove 'static'
- rename to 'Role'

Reply:
If rename to "Role" then will be a conflict with Issue.Role, is this ok?

Reply:
I don't see Issue.Role, where it's located?

Reply:
Sorry, I was wrong, I meant Entities.Role

Reply:
Rename RoleConsts to RoleNames